Make fails

`Make` fails because `browserslist-config-nextcloud` is required but never requested:
```
ERROR in ./src/main.js
Module build failed (from ./node_modules/babel-loader/lib/index.js):
Error: [BABEL] /var/www/nextcloud/apps/tasks/src/main.js: Cannot find module 'browserslist-config-nextcloud' (While processing: "/var/www/nextcloud/apps/tasks/node_modules/@babel/preset-env/lib/index.js")
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:636:15)
    at Function.resolve (/var/www/nextcloud/apps/tasks/node_modules/v8-compile-cache/v8-compile-cache.js:166:23)
    at Object.loadQueries (/var/www/nextcloud/apps/tasks/node_modules/browserslist/node.js:116:35)
    at Function.select (/var/www/nextcloud/apps/tasks/node_modules/browserslist/index.js:951:26)
    at /var/www/nextcloud/apps/tasks/node_modules/browserslist/index.js:255:33
    at Array.reduce (<anonymous>)
    at resolve (/var/www/nextcloud/apps/tasks/node_modules/browserslist/index.js:237:18)
    at browserslist (/var/www/nextcloud/apps/tasks/node_modules/browserslist/index.js:361:16)
    at getTargets (/var/www/nextcloud/apps/tasks/node_modules/@babel/preset-env/lib/targets-parser.js:204:50)
    at _default (/var/www/nextcloud/apps/tasks/node_modules/@babel/preset-env/lib/index.js:132:46)
    at /var/www/nextcloud/apps/tasks/node_modules/@babel/helper-plugin-utils/lib/index.js:19:12
    at loadDescriptor (/var/www/nextcloud/apps/tasks/node_modules/@babel/core/lib/config/full.js:165:14)
    at cachedFunction (/var/www/nextcloud/apps/tasks/node_modules/@babel/core/lib/config/caching.js:33:19)
    at loadPresetDescriptor (/var/www/nextcloud/apps/tasks/node_modules/@babel/core/lib/config/full.js:235:63)
    at config.presets.reduce (/var/www/nextcloud/apps/tasks/node_modules/@babel/core/lib/config/full.js:77:21)
    at Array.reduce (<anonymous>)
    at recurseDescriptors (/var/www/nextcloud/apps/tasks/node_modules/@babel/core/lib/config/full.js:74:38)
    at loadFullConfig (/var/www/nextcloud/apps/tasks/node_modules/@babel/core/lib/config/full.js:108:6)
    at process.nextTick (/var/www/nextcloud/apps/tasks/node_modules/@babel/core/lib/transform.js:28:33)
    at process._tickCallback (internal/process/next_tick.js:61:11)

```

The Tasks app itself does not need this package, so I suspect `nextcloud-vue` or one of the packages used there needs this package but never installs it (it misses in `package.json`). Maybe @skjnldsv or @ChristophWurst could have a look.

The issue was originally reported by @jakobroehrl by email. I couldn't reproduce it first, but after one of the latest package bumps it happens on my machine too.

--- a/.babelrc.js
+++ b/.babelrc.js
@@ -5,12 +5,7 @@ module.exports = {
 	],
 	presets: [
 		[
-			'@babel/preset-env',
-			{
-				targets: {
-					browsers: ['last 2 versions', 'ie >= 11']
-				}
-			}
+			'@babel/preset-env'
 		]
 	]
 }

--- a/package.json
+++ b/package.json
@@ -57,6 +57,7 @@
 		"babel-eslint": "^10.0.3",
 		"babel-jest": "^24.9.0",
 		"babel-loader": "^8.0.6",
+		"browserslist-config-nextcloud": "0.0.1",
 		"css-loader": "^3.2.0",
 		"eslint": "^6.3.0",
 		"eslint-config-standard": "^14.1.0",
@@ -72,8 +73,8 @@
 		"jest-serializer-vue": "^2.0.2",
 		"jsdom": "^15.1.1",
 		"jsdom-global": "^3.0.2",
-		"node-sass": "^4.12.0",
 		"moment": "^2.24.0",
+		"node-sass": "^4.12.0",
 		"prettier-eslint": "^9.0.0",
 		"raw-loader": "^3.1.0",
 		"sass-loader": "^8.0.0",
@@ -122,5 +123,8 @@
 			"lcov",
 			"clover"
 		]
-	}
+	},
+	"browserslist": [
+		"extends browserslist-config-nextcloud"
+	]
 }

